lib/bson/binary.rb

```diff
@@ -39,6 +39,7 @@ class Binary
       :uuid_old => 3.chr,
       :uuid => 4.chr,
       :md5 => 5.chr,
+      :cyphertext => 6.chr,
```

Comment:
"Ciphertext" is spelled with "i".

Reply:
😅 Turns out I don't know how to spell. Thank you.